### PR TITLE
Add com.daniel15.wcc

### PR DESCRIPTION
**Pull request by Daniel15**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly. *WebCamControl allows webcam controls such as pan/tilt/zoom to be adjusted*
- [x] The domain used for the application ID is [controlled by the application developer(s)][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor to the project.

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: com.daniel15.wcc.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      env:
+        - RUNTIME=linux-arm64
+    x86_64:
+      env:
+        - RUNTIME=linux-x64
+
+command: webcamcontrol
+
+finish-args:
+  - --device=dri
```
Comment (hfiguiere) [inline com.daniel15.wcc.yaml:22]: this is redundant with `all`

Comment (Daniel15) [reply com.daniel15.wcc.yaml:22]: Thanks - will remove.

**Comment by Daniel15** [comment]
bot, build com.daniel15.wcc

**Comment by Daniel15** [comment]
bot, build com.daniel15.wcc

File: com.daniel15.wcc.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+id: com.daniel15.wcc
```
[the remark this answers is not in the document]
Comment (Daniel15) [reply com.daniel15.wcc.yaml:1]: Yes, I own both d.sb and daniel15.com. daniel15.com always redirects at the moment: https://github.com/Daniel15/Website/blob/6f5b8948a0bef7cde24197fd0a08b94b8a96d485/Daniel15.Web/nginx.conf#L18

It's late here (San Francisco Bay Area) but I can add a .well-known file tomorrow.

[the remark this answers is not in the document]
Comment (bbhtt) [reply com.daniel15.wcc.yaml:1]: No need to do it now, once the submission is accepted you can verify it to get the blue tick by logging in to flathub.org.

**Comment by bbhtt** [comment]
I don't have a webcam and it shows an error dialogue on my desktop. Can you show a screen record (10-15s)/screenshots of how the app works?

**Comment by Daniel15** [comment]
@bbhtt I'll record a video soon - might be easier to explain the app with a video vs with screenshots.

**Comment by Daniel15** [comment]
@bbhtt Here's a basic video: https://www.youtube.com/watch?v=CDdcrBELXmY
I also created https://daniel15.com/.well-known/github.txt to demonstrate ownership of the domain.

**Comment by bbhtt** [comment]
Thanks

**Comment by bbhtt** [comment]
@Daniel15 GNOME GitLab is down at the moment so the official build from the merge failed. You have to wait until that's fixed and then restart the build https://docs.flathub.org/docs/for-app-authors/maintenance#official-builds

Once you get a successful build it will be published.

**Comment by Daniel15** [comment]
Thanks for your help @bbhtt! Have a great weekend.

**Pull request closed without merging**